aries-site/src/data/structure.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable max-len */
+import {
+  IconComponents,
+  IconDesign,
+  IconDevelop,
+  IconFoundation,
+  IconGuidelines,
+  IconResources,
+  IconBrand,
+  IconColor,
+  IconIcons,
+  IconLayout,
+  IconType,
+  IconToken,
+} from '../components/icons';
+
+export const structure = [
+  {
+    name: 'Guidelines',
+    color: 'green',
+    description:
+      'Why does HPE have a design system? All the aesthetics, best practices, and information about the platform and how to wield it.',
+    icon: size => <IconGuidelines size={size} />,
+    pages: ['Principles', 'Human Centered', 'Philosophy'],
```

Comment:
Recommend adding `parent: 'Home` per this filter comment: https://github.com/hpe-design/aries/pull/168/files?file-filters%5B%5D=.js&hide-deleted-files=true#r363999070